plugin/inline_edit.vim

```diff
@@ -128,6 +128,13 @@ call add(g:inline_edit_patterns, {
       \ 'indent_based':  1,
       \ })
 
+call add(g:inline_edit_patterns, {
+      \ 'main_filetype': 'typescript',
+      \ 'sub_filetype':  'html',
+      \ 'start':         '^\s*template:\s*`',
+      \ 'end':           '`,'
```

Comment:
The trailing comma is probably not mandatory, so maybe a pattern like this would work regardless of whether it's there or if it's the end of the line?
```suggestion
      \ 'end':           '`\(,\|$\)'
```